From cbb4595084b6d31074edc344d4f2aaa9f3b44579 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 10 Apr 2019 13:58:51 +0800
Subject: [PATCH 123/147] arm64: allwinner: a64: pinetab: add accel sensor
 mount matrix

The accel sensor installation orientation is different to the screen
orientation on the PineTab.

Add a accel sensor mount matrix to calibrate it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
index 47c9606b171d..f2641a426fee 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
@@ -184,6 +184,9 @@
 		reg = <0x18>;
 		interrupt-parent = <&pio>;
 		interrupts = <7 5 IRQ_TYPE_LEVEL_HIGH>; /* PH5 */
+		mount-matrix = "0", "-1", "0",
+			       "-1", "0", "0",
+			       "0", "0", "-1";
 	};
 };
 
-- 
2.21.0

